src/script/page/RightSidebar/GuestServicesOptions/components/GuestOptions/GuestOptions.tsx

```diff
@@ -60,30 +71,51 @@ const GuestOptions: FC<GuestOptionsProps> = ({
   isRequestOngoing = false,
   isTeamStateGuestLinkEnabled = false,
   isToggleDisabled = false,
+  core = container.resolve(Core),
 }) => {
   const [isLinkCopied, setIsLinkCopied] = useState<boolean>(false);
   const [conversationHasGuestLinkEnabled, setConversationHasGuestLinkEnabled] = useState<boolean>(false);
-
-  const {accessCode, hasGuest, inTeam, isGuestAndServicesRoom, isGuestRoom, isServicesRoom} = useKoSubscribableChildren(
-    activeConversation,
-    ['accessCode', 'hasGuest', 'inTeam', 'isGuestAndServicesRoom', 'isGuestRoom', 'isServicesRoom'],
+  const [optionPasswordSecured, setOptionPasswordSecured] = useState<PasswordPreference>(
```

Comment:
Since there are only 2 preclusive values, wouldn't it be easier to store a `isPasswordSecurde` boolean value instead?

Reply:
in the design it is a Radio that's why I went with an enum:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/63250054/234518194-613d5c4a-bb50-4e97-8ee9-c5fe479ef1e4.png">

Reply:
Ok, makes sense, then I'm fine with enum :D